lib/src/bloc/i_movies_bloc.dart

```diff
@@ -7,4 +7,5 @@ abstract class IMoviesBloc extends Bloc {
   Stream<MoviesModel> get stream;
 
   void fetchMovies();
+  void fetchMovieSearch(String movieKeywords);
```

Comment:
You can changed this function name with a more descriptive, in the first case if you are fetching trending movies, you can call fetchTrendingMovies, the second one is better to call fetchMoviesBySearch

Reply:
Done